CONTRIBUTING.md

```diff
@@ -130,6 +130,8 @@ It is recommended to use it over the built-in Lua plugins.
 Please note that EmmyLua is not available for other editors based on Visual Studio Code,
 such as [VSCodium](https://vscodium.com) or [Eclipse Theia](https://theia-ide.org) but can be built from source if needed.
 
+Note that you will need to have java installed to use emmyLua, and have the JAVA_HOME environment variable correctly setup.
```

Comment:
The environment variable wouldn't work for me in VSCode, so I ended up just adding this to my `settings.json` :
```json
"emmylua.java.home": "C:/Program Files (x86)/Java/jre1.8.0_201/",
```